### DISCUSSION TITLE
Victory Native - Pie Chart - How to position labels in center hole

### DISCUSSION THREAD

**Discussion by mdineshari** [FormidableLabs/victory#2112] [Q&A]
Hello, 

I wanted to achieve three use cases which i tried different approaches and failed to achieve result i expect. I would like to get some help for you guys suggestion to achieve it.

1.) How to render legends in the center of the donut hole in responsive way?
2.) How to render labels inside each pies at center position?
3.) How to click single pie with animation, box shadow also open custom component in native element?

![image](https://user-images.githubusercontent.com/13847501/152691043-efa6f54c-4646-4a29-8fbf-707c895ea6b1.png)

**Comment by becca-bailey** [comment]
We are moving our Q&A issues to discussion threads, so I will move this one over!
